.solhintignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 artifacts
 contracts/smart-account/test
-contracts/smart-account/libs
\ No newline at end of file
+contracts/smart-account/libs
+contracts/smart-account/modules/AccountRecoveryModule.sol
```

Comment:
we plan to fix linting soon for these contracts? in different PR?

Reply:
>AccountRecoveryModule.sol
it was added here because it violates the rule about function max length in lines and cyclomatic complexity.
All other linting is in place.
`validateUserOp` is longer than expected, but for the sake of gas savings decided to keep it as is. Check [this](https://biconomyworkspace.slack.com/archives/C05E6G3NTAN/p1697098546139319) thread for more details